plugins/mumble_plugin_win32.h

```diff
@@ -18,9 +18,24 @@
 
 #include "mumble_plugin.h"
 
+#if _MSC_VER < 1800 && defined(M_IX86)
+typedef unsigned long long uintptr64_t;
+typedef unsigned long uintptr32_t;
+#else
+#include <stdint.h>
+typedef uint64_t uintptr64_t;
```

Comment:
I get your point about them not being "pointers", but rather... "pointers in another process".

So... I take my comment back. They aren't really "just a pointer".